style.css

```diff
@@ -0,0 +1,127 @@
+.slideShow-container, .reference, .slides-container, .btn, .Option-bar, .counter, .imgs, .imgs>img, .thumbnails-popUp, .title{
```

Comment:
Let's have this like this

```css
.slideShow-container,
.slideShow-container,
.slides-container,
.btn,
.Option-bar,
.imgs,
.imgs > img,
.thumbnails-popUp,
.title
```

why do you think this could be helpful?

Reply:
⚠️  .Option-bar breaks your naming convention

Reply:
For ones, it is more easy to read.